### PR TITLE
Add a job to produce coverage information

### PR DESCRIPTION
**Pull request by Katharine**
This adds a periodic job that produces (but does not diff) coverage information. Hopefully.

I have absolutely no idea whether or not it actually works. ¯\\\_(ツ)\_/¯

Depends on #9370.

/cc @BenTheElder

### CODE REVIEW

**Comment by BenTheElder** [comment]
Er, you'll need to add a matching entry in the testgrid config

**Comment by Katharine** [comment]
Now with added testgrid, possibly.

image

File: config/jobs/kubernetes/sig-testing/coverage.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20180829-29e661965-master
+      command:
+      - bash
+      args:
+      - -e
+      - -c
+      - |
+        shopt -s globstar
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-testing/coverage.yaml]: oh this is uh.. ok

Comment (stevekuznetsov) [reply config/jobs/kubernetes/sig-testing/coverage.yaml]: Someone will regret this in the future :)

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-testing/coverage.yaml]: yeah... 
Ideally we're going to replace this when we get everything out of bootstrap / scenarios / ... Sen and I would really like to get a better kubetest and better patterns for test scripts without any of the existing junk in the not terribly distant future... 

of course more likely we won't get to that and inline bash in pods will continue to be all over kubernetes testing /shrug

**Comment by spiffxp** [comment]
/lgtm
/hold cancel
>I have absolutely no idea whether or not it actually works. ¯\_(ツ)_/¯
let's find out

**Pull request merged**